lib/ui/src/components/preview/FramesRenderer.tsx

```diff
@@ -72,6 +90,9 @@ export const FramesRenderer: FunctionComponent<FramesRendererProps> = ({
   return (
     <Fragment>
       <Global styles={styles} />
+      <SkipToSidebarLink secondary isLink tabIndex={0} href={`#${storyId}`}>
+        Skip to sidebar
+      </SkipToSidebarLink>
```

Comment:
This button should not be rendered when in `singleStory` mode. In this case there is no sidebar: `showNav` is always `false`. You can enable this mode by passing `singleStory=true` as a query param.

It's also possible the sidebar is hidden (e.g. in fullscreen mode, or by pressing `S`). We should then either enable the sidebar when pressing the button, or not render the button at all (which would also cover singleStory mode). I'm fine with either approach. @domyen which you do you think makes sense here?

See [`lib/api/src/modules/layout.ts`](https://github.com/storybookjs/storybook/blob/next/lib/api/src/modules/layout.ts).

Reply:
Excellent catch, @ghengeveld 💯

Reply:
I believe I've fixed this, but feel free to unresolve if the issue persists 👍